### PR TITLE
Fix MGLOfflineStorageTests

### PR DESCRIPTION
**Pull request by birkskyum**
This invalidates MGLOfflinePack properly

Screen Shot 2022-09-03 at 19 28 57

### CODE REVIEW

**Review by roblabs — COMMENTED** [review]
There is only a manual test for Offline Packs in the scheme `iosapp`.  Can you verify that it still works?

**Comment by birkskyum** [comment]
It should be able to pass now, so we can turn it back on. Where is it disabled?

**Comment by roblabs** [comment]
It's not disabled, but manually run via the scheme `iosapp`.

> @birkskyum, "*It should be able to pass now, so we can turn it back on to run automatically. Where is it disabled?*"

**Comment by birkskyum** [comment]
From what I can tell it's running just fine now

Screen Shot 2022-09-07 at 16 25 11

File: platform/ios/platform/ios/test/CI.xctestplan [hunk 1]
```diff
@@ -31,8 +31,7 @@
   "testTargets" : [
     {
       "skippedTests" : [
-        "MGLMapSnapshotterTests\/testLocalGlyphRendering",
-        "MGLOfflineStorageTests\/testAddPackForBounds"
+        "MGLMapSnapshotterTests\/testLocalGlyphRendering"
```
Comment (roblabs) [inline platform/ios/platform/ios/test/CI.xctestplan:34]: Excellent.  Testing in the `CI` scheme works as well.  thanks for enabling the disabled tests.

**Review by roblabs — APPROVED** [review]

**Pull request merged**